internal/config.go

```diff
@@ -206,7 +210,7 @@ func (c *Config) Validate() {
 		log.Infoln("for better security, \"secret\" should ideally be 32 bytes or longer")
 	}
 
-	if c.ProviderUri == "" || c.ClientId == "" || c.ClientSecret == "" {
+	if c.ProviderURI == "" || c.ClientID == "" || c.ClientSecret == "" {
```

Comment:
As the `EncryptionKeyString` is now required, I'd suggest to add a validation step for it here.